pyspark/dl/optim/optimizer.py

```diff
@@ -172,7 +346,8 @@ def __init__(self,
        :param batch_size: training batch size
        """ 
        JavaValue.__init__(self, None, bigdl_type, model.value,
-                           training_rdd, criterion, optim_method,
+                           training_rdd, criterion,
+                          optim_method if optim_method else SGD(),
```

Comment:
Add a doc to describe this default behavior?
optim_method = SGD() if optim_method is None else optim_method